docs/deployment/cumulus_api_launchpad_authentication.md

```diff
@@ -23,4 +24,6 @@ LAUNCHPAD_PASSPHRASE=<LAUNCHPAD_PASSPHRASE>
 
 3. Configure the `oauth.provider` and `oath.userGroup` configuration parameters in `app/config.yml`. _Example configuration of the `oauth` variables can be found in Cumulus core's [example](https://github.com/nasa/cumulus/blob/master/example/app/config.yml)_
 
-`oauth.provider` in `app/config.yml` must be set to `launchpad` to use Launchpad authentication. 
+`oauth.provider` in `app/config.yml` must be set to `launchpad` to use Launchpad authentication.
+
+[^1]: Authentication by bringing your own launchpad token returned from the NASA launchpad `/gettoken` endpoint
```

Comment:
Curious: why is this a footnote rather being inline? Also, launchpad should be consistently capitalized to Launchpad

Reply:
No good reason I guess.